### ISSUE TITLE
Clean install not working due to adding default data

### ISSUE THREAD

**Issue by WHITE-developer** [verbb/wishlist#43]
### Description
When using existing project.yaml to create a fresh project, you get an error now that the lists configuration is saved in there.  
Best should be to not save them to the project yaml. Other thing is that you can access the "add new lists" when the "allowAdminChanges" is set to false, although you can't save it in the end?

### Steps to reproduce

1. Get an project.yaml where the wishlist default data is available
2. Use that to install a fresh database

### Additional info

- Plugin version: 1.1.2
- Craft version: 3.3.20

**Comment by engram-design** [comment]
What's the specific error you get when you try to apply the project config on your fresh site? I've tested this locally, and haven't got errors. It should only be creating new list types as per your source site, because they're stored in the project config.

Lists themselves aren't stored in project config, and as such, you can certainly add new lists when `allowAdminChanges` is set to `false`. This is important, because new lists are created every time a user or guests adds things to their wishlist for the first time, the new list needs to be created.

List types are the only thing stored in project config.

**Comment by WHITE-developer** [comment]
We have 2 wishlist types, "Last Viewed Products" and "Wishlist", both became saved in the project.yaml:
```
wishlist:
  listTypes:
    557ff8bb-49f7-4008-90d5-6e63165f23c3:
      name: 'Last Viewed Products'
      handle: lastViewedProducts
      default: '0'
      listFieldLayouts:
        f6b82fcc-0def-4673-b4e6-5886002e4237: null
      itemFieldLayouts:
        516e5e89-4b19-4c29-949f-a14c27801e77: null
    9dfd4666-a42e-4ddc-80e8-966b52d018d0:
      name: Wishlist
      handle: wishlist
      default: '1'
      listFieldLayouts:
        9ffc1a20-8a0e-4b32-9987-f7ecdd7d99a0: null
      itemFieldLayouts:
        9948eef9-f718-4d05-92e2-4d4c7049a6eb: null
```
Trying to install from scratch with that in the yaml I get
```
Next craft\errors\MigrationException: An error occurred while executing the "craft\migrations\Install migration: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'wishlist' for key 'craft_wishlist_listtypes_handle_unq_idx'
The SQL being executed was: INSERT INTO `craft_wishlist_listtypes` (`uid`, `name`, `handle`, `default`, `dateCreated`, `dateUpdated`) VALUES ('9dfd4666-a42e-4ddc-80e8-966b52d018d0', 'Wishlist', 'wishlist', 1, '2020-04-01 07:46:00', '2020-04-01 07:46:00') in /var/www/html/vendor/craftcms/cms/src/db/MigrationManager.php:256
```
Commenting out line 30 in the `Install.php` migration works fine and does install.

With `allowAdminChanges` to `false`, you are still able to access the "List Types" page and trying to create a new List type gives of course `Not Supported – yii\base\NotSupportedException
Changes to the project config are not possible while in read-only mode.`

**Comment by engram-design** [comment]
Gotcha - fixed in [1.2.2](https://github.com/verbb/wishlist/releases/tag/1.2.2)
